test/patch/registerElement.spec.js

```diff
@@ -129,4 +129,27 @@ describe('document.registerElement', ifEnvSupports(registerElement, function ()
     var elt = document.createElement('x-props-desc');
   });
 
+
+  it('should check bind callback if not own property', function (done) {
+    testZone.run(function() {
+      var originalProto = {
+        createdCallback: function() {}
+      }
```

Comment:
Could you add an assertion in the createdCallback:
```
assert(window.zone).toBeDirectChildOf(testZone);
done();
```

and create the custom element at the end of the test (out of the testZone)